apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/workingcopies/WorkingcopiesWebscript1.java

```diff
@@ -108,14 +108,21 @@ public void cancelWorkingcopy(@UriVariable final String space, @UriVariable fina
 
     @ApiOperation("Returns the original node for given working copy")
     @Uri(value = "/workingcopies/{space}/{store}/{guid}/original", method = HttpMethod.GET)
-    @ApiResponses(@ApiResponse(code = 200, message = "Success", response = NoderefResult.class))
+    @ApiResponses({
+            @ApiResponse(code = 200, message = "Success", response = NoderefResult.class),
+            @ApiResponse(code = 404, message = "Not Found")
+    })
     public void getWorkingCopySource(@UriVariable final String space, @UriVariable final String store,
             @UriVariable final String guid,
             WebScriptResponse response) throws IOException {
         NodeRef workingCopyRef = this.createNodeRef(space, store, guid);
-        NodeRef originalRef = this.nodeService.getWorkingCopySource(workingCopyRef);
-
-        writeJsonResponse(response, new NoderefResult(originalRef));
+        if(this.nodeService.exists(workingCopyRef)) {
+            NodeRef originalRef = this.nodeService.getWorkingCopySource(workingCopyRef);
```

Comment:
Stylistic nitpick: There's 3 instances of unnecessary `this.` prefixes on these lines.